Can't run unit test after upgrade

**Describe the bug**
After I upgraded the koin test deps to the latest stable version I can't run the unit-test again.

kotlin 1.8.0
gradle 7.3.1

```
koinTest      : "io.insert-koin:koin-test:3.3.2",
koinTestJunit4: "io.insert-koin:koin-test-junit4:3.3.2",
koinTestJunit5: "io.insert-koin:koin-test-junit5:3.3.2",
```

> FAILURE: Build failed with an exception.
> * What went wrong:
> Execution failed for task ':app:kaptGenerateStubsDevDebugUnitTestKotlin'.
> > Could not resolve all files for configuration ':app:devDebugUnitTestCompileClasspath'.
>    > Could not resolve org.jetbrains.kotlin:kotlin-test-junit5:1.8.0.
>      Required by:
>          project :app
>          project :app > org.jetbrains.kotlin:kotlin-test:1.8.0
>       > Module 'org.jetbrains.kotlin:kotlin-test-junit5' has been rejected:
>            Cannot select module with conflict on capability 'org.jetbrains.kotlin:kotlin-test-framework-impl:1.8.0' also provided by [org.jetbrains.kotlin:kotlin-test-junit:1.7.21(junitApi)]
>    > Could not resolve org.jetbrains.kotlin:kotlin-test-junit:1.7.21.
>      Required by:
>          project :app > io.insert-koin:koin-test-jvm:3.3.2
>       > Module 'org.jetbrains.kotlin:kotlin-test-junit' has been rejected:
>            Cannot select module with conflict on capability 'org.jetbrains.kotlin:kotlin-test-framework-impl:1.7.21' also provided by [org.jetbrains.kotlin:kotlin-test-junit5:1.8.0(junit5Api)]
>

--- a/projects/core/koin-test-junit4/build.gradle.kts
+++ b/projects/core/koin-test-junit4/build.gradle.kts
@@ -7,7 +7,7 @@ plugins {
 dependencies {
     api(project(":core:koin-test"))
     implementation(libs.test.junit)
-    implementation(libs.kotlin.test)
+    testImplementation(libs.kotlin.test)
     testImplementation(libs.test.mockito)
 }
 

--- a/projects/core/koin-test/build.gradle.kts
+++ b/projects/core/koin-test/build.gradle.kts
@@ -34,7 +34,6 @@ kotlin {
     sourceSets {
         commonMain.dependencies {
             api(project(":core:koin-core"))
-            implementation(libs.kotlin.test)
         }
         commonTest.dependencies {
             implementation(libs.kotlin.test)

--- a/projects/test-macos.sh
+++ b/projects/test-macos.sh
@@ -1,0 +1,3 @@
+#!/bin/sh
+
+./gradlew cleanTest :core:koin-core:macosArm64Test --parallel --no-build-cache

